src/corpus/domain/LineVariant.test.ts

```diff
@@ -1,85 +1,22 @@
+import {
```

Comment:
The test tokens were all very similar. I generalized them into a number of test cases and extracted them into a separate file.